### ISSUE TITLE
Composition Tab

### ISSUE THREAD

**Issue by gfrc11752** [Huh/PopR_SDGFP#68]
I am having troubles uploading WTD composition data. Receiving a java script error message.

**Comment by paul-lukacs** [comment]
It seems to be something with the data download regardless of species.   Upon clicking ‘get data’, download begins then the following error message appears.

[cid:image001.png@01D125D2.8B980250]

**Comment by Huh** [comment]
I believe the issue is actually with the server that the DAU page is connecting with for the maps.  It seems to be giving bad responses that are killing our server.  I would like to give it a little time to see if the other server is down.  I will check again in an hour.

**Comment by Huh** [comment]
I finally found the error(s).  The problem was that geos (a bit of underlying softwareused for spatial manipulations) was crashing R.  The geos problem was a known error and I had to reinstall from source, so it took a bit.  Then gdal and proj couldn't talk to each other so they got uninstalled and reinstalled.  Round and round it went working out all the dependencies and eventually it worked again.  I don't know if I upgraded something or if this was the result of a server wide update.  

Thanks for pointing this out Kevin, it definitely needed to get tracked down because it was breaking everything.  Tricky though because it worked on windows, just not linux and then it worked on linux and not on the web.  

I think we got it now, let me know if you experience anymore issues.

The DAU tab is updated for the admin login.
